routemaster/client.rb

```diff
@@ -204,15 +179,15 @@ def _assert_valid_data(value)
         raise InvalidArgumentError, e
       end
 
-      def _send_event(event, topic, callback, t: nil, async: false, data: nil)
+      def _send_event(event, topic, callback, t: nil, async: false, data: nil, target: nil)
         _assert_valid_url!(callback)
         _assert_valid_topic!(topic)
         _assert_valid_timestamp!(t) if t
         _assert_valid_data(data) if data
```

Comment:
does it worth validating the `target` here? Do we enforce any particular format for subscription names in the world of routemaster?

Reply:
I'd say check it's an string matching `/\A[A-Za-z0-9_-]+\Z/`? Or event, just that it's a string. The server would yell anyhow if it's incorrect.